docs/general/metrics.md

```diff
@@ -125,6 +126,22 @@ to the processes then to represent the count of the processes we can have a metr
 `system.processes.count`. The suffix `count` here indicates that it is the count of
 `system.processes`.
 
+This rule SHOULD only be applied to UpDownCounters, since (monotonic) Counters have
+`_total` appended to their names when they are
+[mapped to Prometheus](https://github.com/open-telemetry/opentelemetry-specification/blob/v1.21.0/specification/compatibility/prometheus_and_openmetrics.md#otlp-metric-points-to-prometheus),
+which would lead to `_count_total`.
+
+#### Do not use `total`
+
+Counters SHOULD NOT append `_total`  to their names. Counters have `_total` appended to their names when they are
+[mapped to Prometheus](https://github.com/open-telemetry/opentelemetry-specification/blob/v1.21.0/specification/compatibility/prometheus_and_openmetrics.md#otlp-metric-points-to-prometheus),
+which would lead to `_total_total`. The reason that the Prometheus mapping cannot
```

Comment:
The [prometheus mapping](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/compatibility/prometheus_and_openmetrics.md#sums) does suppress duplicate `_total`: `If the metric name for monotonic Sum metric points does not end in a suffix of _total a suffix of _total MUST be added by default`.

It isn't fully bidirectional.  When converting prom -> OTLP, we leave the `_total` suffix intact by default per [The metadata spec](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/compatibility/prometheus_and_openmetrics.md#metric-metadata): `The OpenMetrics MetricFamily Name MUST be added as the Name of the OTLP metric. By default, the name MUST be unaltered, but translation SHOULD provide configuration which, when enabled, removes type (e.g. _total) and unit (e.g. _seconds) suffixes.`

Reply:
thanks, I've removed this part